hello-world-notification-microservice/README.md

```diff
@@ -72,4 +76,69 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+    - for free trial account on cumulocity.com you will need to request this feature by contacting
+- contents of this repository
+
+### Instructions
+1. Creating a test user
+    - From the Application Switcher select Administration
+    - In the Navigation pane expand Accounts and go to Users
+    - Click on 'Add user', fill in the form for the new user, untick "Send password reset link as email" and provide a password
+    - Click Save
+    - Enable notification support:
+        - Using Navigation pane go to Accounts and then Roles, click on "Add global role"
+        - Name the role "Notifications" and from the list of permissions tick "Notification 2" in Admin column
+        - click Save
+        - go to Accounts then Users
+        - in the list find the username created a few steps before and under "Global roles" open the dropdown
```

Comment:
I think the line above requires a comma after 'in the list' OR write the other way round without a comma (I prefer with a comma for this).
 The other way round is 'find the username created a few steps before in the list and under "Global roles" ...' (I prefer the additional comma approach - more natural)